reference/CloudPay-API-Specification.yaml

```diff
@@ -1381,6 +1381,115 @@ paths:
 
         **Note** *Only one of the four authorisation parameters are required*
     parameters: []
+  /api/v1/account/profile/billing:
+    put:
+      summary: Update the customer's Stripe billing profile
+      operationId: put-api-v1-account-profile-billing
+      parameters:
+        - $ref: '#/components/parameters/apiJwtTokenParam'
+        - $ref: '#/components/parameters/apiSessionIdParam'
+        - $ref: '#/components/parameters/thirdPartyAuthCookie'
+        - $ref: '#/components/parameters/authCookie'
+      tags:
+        - Account
+        - Profile
+        - Billing details
+      responses:
+        '202':
+          description: Accepted
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '403':
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '500':
+          description: Error Response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+      description: |-
+        ![In Review](https://img.shields.io/badge/Status-In%20Review-green?style=flat-square)
+        The update customer billing profile endpoint allows a customer to update their billing details.
+        Please be aware that the customer billing profile depends on Provider - Stripe or Paypal.
+        **Note** *Only one of the four authorisation parameters are required*
+      requestBody:
+        content:
+          application/json:
+            schema:
+              description: 'Customer Stripe billing profile'
+              type: object
+              properties:
+                  AddressCountry:
+                    type: string
+                  AddressCity:
+                    type: string
+                  AddressLine1:
+                    type: string
+                  AddressLine2:
+                    type: string
+                  AddressState:
+                    type: string
+                  AddressZip:
+                    type: string
+                  CardDetails:
+                    type: object
```

Comment:
Does this need to be an array?

Reply:
Not an array - only one card details per customer because one billing profile is assigned. This is a comment for new feature development - customer can have more then one card saved. :)